packages/storybook/stories/components/flag/migration.from.17.x.mdx

```diff
@@ -0,0 +1,63 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="ODS Components/Content/Flag/Migration From 17.x" />
+
+# Flag - migrate from v17 to v18
+----
+
+Flag has been removed from ODS components.
+
+Using ODS Flag, pre V18, does require every project to setup a specific build configuration to be able to fetch the assets.
+
+This whole config should not be enforced by ODS and should be manage by each the way they prefer.
+
+## The assets issue
+
+Due to how Stencil works, there is no way to lazy load specific assets on demand.
+
+Possible solutions are:
+- inlining the assets in the final bundle
+- externalize the assets to a remote source
+- setup the assets path on each project using the web-component
+
+### Inlining assets
+
+This solution will transform the assets into base64 strings and embed them in the final web-component bundle.
+
+It is fine when dealing with small assets (like the spinner svg).
+
+But, flags are high quality svg that add way to much weight to the bundle, thus it is not a viable solution.
```

Comment:
However, flags are usually high-quality svg files that will add weight to the bundle, thus it is not a viable solution.